### DISCUSSION TITLE
I would like to be able to have a quick search box to find an action by name

### DISCUSSION THREAD

**Discussion by philippemiguet** [community/community#76406] [Actions]
### Select Topic Area

Product Feedback

### Body

We have a lot of actions on our mono-repo, so finding the right action to trigger manually is painful as we have to "load more" several times.

If we could have a small search box to quickly filter out actions that would be amazing!

Ideally, this search box could be populated from a URL query string too so from our GH template we can pre-filter the actions on the page for our engineers.

**Comment by adeturner** [comment]
Same here - a filter search for actions would be a great enhancement

**Comment by adeturner** [comment]
I noticed a solution was added to the public roadmap two weeks back:
https://github.com/orgs/github/projects/4247/views/1?filterQuery=is%3Aopen+-status%3A%22Q4+2022+%E2%80%93+Oct-Dec%22%2C%22Q2+2023+%E2%80%93+Apr-Jun%22++workflow&pane=issue&itemId=48129706
